### ISSUE TITLE
Increase in FA icon size 3.2.0 to 3.2.1

### ISSUE THREAD

**Issue by Finii** [ryanoasis/nerd-fonts#1614]
Apologies if this is unrelated or intended, but I noticed an **increase** in the size of some symbols in `ttf-firacode-nerd`, going from `3.2.0` -> `3.2.1` in my i3 status bar.

| `3.2.0` | `3.2.1` |
| --- | --- |
|![2024-04-18_19-08](https://github.com/ryanoasis/nerd-fonts/assets/3248296/456c0092-613c-4197-b69e-bfd508123448)|![2024-04-18_19-09](https://github.com/ryanoasis/nerd-fonts/assets/3248296/feb63bf6-b3c7-496e-863d-f62982282262)|

`U+F120` (`nf-fa-terminal`) and `U+F11B` (`nf-fa-gamepad`)

_Originally posted by @martinpelikan in https://github.com/ryanoasis/nerd-fonts/issues/1611#issuecomment-2065636039_

**Comment by Finii** [comment]
A lot glyphs changed size, but why? :grimacing: :laughing: 

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/cb9b9979-b128-4750-ad39-0538f818701c)

**Comment by Finii** [comment]
Ah here, it seems to be intended

1fe58378a2a3890a413b06830ce857d203296207  `FontAwesome: Scale glyphs a bit bigger`

Which is part of
* #1596 

Lets compare the glyphs

You see, a lot glyphs got smaller on `v3.1.1` -> `v3.2.0` and grew back to a comparable size on `v3.2.1`.

For some other glyphs they stayed the same from `v3.1.1` to `v3.2.0` and now only grew with `v3.2.1`. But they usually also changed the outline (i.e. look different anyhow).

I don't know. The icons all changed with the substantial update from `FA 4.7` (2016) to `FA 6.5.1` (2024). The relative to each other sizes changed. It is hard to know what is now 'right'. In total the FA of our `v3.2.0` was a bit on the small size (got reported) - not for all but for a lot icons. That was the reason to lift the size a bit.

Do you think the icons are now too big?
Sizewise I would look at `v3.1.1` and `v3.2.1`, as `v3.2.0` just lived 2 weeks or so.

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/e342896e-6ae9-47dc-94e8-35617206dca8)

**Comment by martinpelikan** [comment]
Thank you for the very quick investigation and detailed response!

> Do you think the icons are now too big?

From the perspective of my tiny use case, yes, but possibly because I grew used to the previous size. I'm not sure I would notice had they not overflowed the fixed size I had configured, something I can easily change.

I think I prefer the previous size because the line weight/thickness appears consistent with other symbols I use, but I haven't compared against many others. I wouldn't push for further changes if others are happy with where things are at.

Here's a comparison of the versions you mentioned:

| `3.1.1` | `3.2.1` |
| --- | --- |
|![2024-04-19_00-44](https://github.com/ryanoasis/nerd-fonts/assets/3248296/b0595a97-2ebd-4b93-a64b-df416101826b)|![2024-04-19_00-45](https://github.com/ryanoasis/nerd-fonts/assets/3248296/2769a7af-3716-415f-9dbf-b9280f6b88dc)|

**Comment by treequin** [comment]
Also speaking from my personal use case, I find the increase in size makes FA icons difficult to integrate with icons from other sets.

I use `U+F015` (`nf-fa-home`) with `U+E725` (`nf-dev-git_branch`) for my prompt, and `U+F057` (`nf-fa-times_circle`) and `U+F05A` (`nf-fa-info_circle`) with `U+F0028` (`nf-md-alert_circle`) and `U+F02D7` (`nf-md-help_circle`) for diagnostics. A before-and-after comparison:

| `3.1.1` | `3.2.1` |
| --- | --- |
|![branch-old](https://github.com/ryanoasis/nerd-fonts/assets/133705172/5d572cdc-b2db-44ef-802d-a0396fb9d30d)|![branch-new](https://github.com/ryanoasis/nerd-fonts/assets/133705172/e1594bd0-355f-4c91-8d73-4bb96203be20) |
| ![diagnostics-old](https://github.com/ryanoasis/nerd-fonts/assets/133705172/049a4884-b7f8-46b7-9b7f-d2101b5fe810) | ![diagnostics-new](https://github.com/ryanoasis/nerd-fonts/assets/133705172/d8c49c33-72ef-4100-8164-b416e00a8c87)|

Personally, I preferred the old size, but if the change was necessary I can switch to icons from other sets without problem.

**Comment by Finii** [comment]
@treequin What are these secondary circles in the bottom picture? They are not important I guess?

I guess this needs are more systematic approach 😬 Thanks for the input.

**Comment by treequin** [comment]
> @treequin What are these secondary circles in the bottom picture? They are not important I guess?

I'm not sure I understand, do you mean the zeros? They're just the number of diagnostics of each type in the document (I took the screenshot on an empty file).

**Comment by Finii** [comment]
> the zeros?

:woman_facepalming: 

Yes. Thanks! :smiley_cat:

**Comment by Bvngee** [comment]
Ah, I'm glad I found this - I was wondering why my diagnostics icons in neovim looked noticeably bigger and worse all of a sudden! I will definitely be reverting to nerd-fonts 3.1.1 for now. As a side effect I also much prefer the smaller, more outlined appearance of nf-fa-house, which I guess was changed in an update of font awesome? or maybe just the increased icon size changed the appearance? Not sure, but I will stick with the smaller one which I've been using on my website for a while.
